Makefile

```diff
@@ -14,10 +14,9 @@ test:
 	go test -cover $(shell go list ./... | grep -v /vendor/)
 
 debug:
-	HUMAN_LOG=1 go run -race cmd/dp-code-list-api/main.go
+	HUMAN_LOG=1 GRAPH_DRIVER=neo4j GRAPH_ADDR="bolt://localhost:7687" go run -race cmd/dp-code-list-api/main.go
```

Comment:
still don't get the env variables not in the environment - will still approve (it works) but would like a chat next week.